sympy/codegen/tests/test_rewriting.py

```diff
@@ -245,3 +246,15 @@ def test_logaddexp2_opt():
     assert logaddexp2(x, y) - opt1 == 0
     assert logaddexp2(y, x) - opt1 == 0
     assert opt1.rewrite(log) == expr1
+
+
+def test_optims_numpy():
+    x = Symbol('x')
+    e1 = sin(x)/x
+    assert optimize(e1, optims_numpy) - sinc(x) == 0
+    e2 = sin(2*x)/(2*x)
+    assert optimize(e2, optims_numpy) - sinc(2*x) == 0
```

Comment:
I think it would be clearer just to write
```python
assert optimize(sin(2*x)/(2*x), optims_numpy) == sinc(2*x)
```
You could also use `opt = lambda e: optimize(e, optims_numpy)` if you want to reduce it to
```python
assert opt(sin(2*x)/(2*x)) == sinc(2*x)
```

Reply:
thanks, I rewrote them even further, let me know what you think.